Tensile/Source/lib/include/Tensile/PropertyMatching.hpp

```diff
@@ -346,6 +346,9 @@ namespace Tensile
                     std::cout << "Considered " << considered << "% of entries." << std::endl;
                 }
 
+                if(T_Debug)
+                    std::cout << "Solution index selected: " << bestMatch->index << std::endl;
```

Comment:
```suggestion
                if(T_Debug && bestMatch)
                    std::cout << "Solution index selected: " << bestMatch->index << std::endl;
```

It is possible for `bestMatch` to be null here.